unified-codes/tools/dockerfiles/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+services:
+  legacy-api:
+    container_name: legacy-api
+    build: 
+      context: ../..
+      dockerfile: tools/dockerfiles/legacy-api.Dockerfile
+    image: unified-codes/legacy-api:latest
+    ports:
+      - "3000:3000"
```

Comment:
Looks like you're port mapping here (better way of doing what I said above, probably?)

Reply:
Yeah - it does what you mentioned above internally, I think (though I didn't explicitly define 127.0.0.1 here, it seems to be the default behaviour).